lnbits/helpers.py

```diff
@@ -206,13 +207,10 @@ def is_valid_external_id(external_id: str) -> bool:
 
 
 def is_valid_pubkey(pubkey: str) -> bool:
```

Comment:
this should not be changed, the function is nostr agnostic
the call to this function should be made with the already normalized pubkey